client/app/components/composites/MenuPriority/MenuPriority.css

```diff
@@ -1,8 +1,18 @@
 .menuPriority {
   display: flex;
   align-items: center;
+
+  @nest :global(.js-enabled) & {
+    visibility: hidden;
+
+    &.isMeasured {
+      visibility: visible;
+    }
+  }
 }
 
+.isMeasured {}
```

Comment:
Yes, it should. I thought I ran the linter without errors, but that clearly shouldn't pass with our config. Thanks for noticing.